### DISCUSSION TITLE
Why are the tags in your template without hashtags?

### DISCUSSION THREAD

**Discussion by iwcvanderlinden** [BryanJenksCommunity/FAQ#35] [Q&A] [answered]
Hi Bryan - thank you for all the great content you've made available!

I downloaded your templates and are implementing a workflow using your suggestions on tags and links. 

I noticed that the emoji tags setup for your templates do not actually include a hashtag. Is this deliberate? It actually seems to result in a nice search experience because it does not clutter the view by highlighting the tag you selected. 

However I cannot reproduce these tags - which is a problem if I want to add more ofc. I have tried several things: 
- Tagging with #emoji results in different behaviour than your tags (in search and graph)
- Tagging without # does not result in a searchable tag (which makes sense, but then why are those from your template searchable?)

The image might clarify further. 

![Screen Shot 2021-03-03 at 20 24 43](https://user-images.githubusercontent.com/3098787/109860531-93994e80-7c5e-11eb-80f7-91355797c625.png)

**Comment by tallguyjenks** [comment, marked as answer]
the emoji's in the yaml headers do not work with the octothorpes `#`

if you were to use a tag like `#✅️`

if you place it in the body of the note it will look like this: 

```markdown
Normal Note text #✅️
```

if you were to place the tag in the yaml header it has to be without the octothorpe like this: 

```yaml
---
tags: ✅️
---
```

If the issue is that when you type `#` you're not getting an auto-complete of any of the emoji's to fill in where you want to place them, what you can do is place all of the tags/emojis in a single note that way Obsidian can "see" them and will autocomplete in the future. I personally just use alfred on mac to play my emoji's, Linux you could use Dmenu, windows you can insert emoji's with `Win+.`

**Reply by iwcvanderlinden** [reply]
> the emoji's in the yaml headers do not work with the octothorpes `#`

Thanks a lot, that's what I needed to know! And once again, I'd like to mention how incredibly useful your tips and resources are.

**Reply by tallguyjenks** [reply]
You're welcome and my pleasure :)
